Bottom: f48e90adb114d1d92b8cb1ba5cc26e22d0eba9c1
Top:    c3730d4d38c1401aef816bd37b4262f00eecfcce
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-20 23:22:00 +0100

hid/common: Draw the hole outline for pins / vias being moved or placed


---

diff --git a/src/crosshair.c b/src/crosshair.c
index 87e45fb..d86a194 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -94,7 +94,7 @@ thindraw_moved_pv (PinType *pv, int x, int y)
   moved_pv.X += x;
   moved_pv.Y += y;
 
-  gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_pv, false, false);
+  gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_pv, true, false);
 }
 
 /* ---------------------------------------------------------------------------
